api/schema_system.go

```diff
@@ -105,6 +105,11 @@ func (policy *SchemaReviewPolicy) Validate() error {
 	return nil
 }
 
+// Empty will check if the SchemaReviewPolicy is empty.
+func (policy *SchemaReviewPolicy) Empty() bool {
```

Comment:
Should this be private method if not used elsewhere? If that's the case, we can inline it in SchemaReviewPolicy's Validate() line 97.